packages/babel-preset-react-app/create.js

```diff
@@ -79,7 +89,15 @@ module.exports = function(api, opts, env) {
         // Latest stable ECMAScript features
         require('@babel/preset-env').default,
         {
-          // Allow importing @babel/polyfill in entrypoint and use browserlist to select polyfills
+          // We want Create React App to be IE 9 compatible until React itself
+          // no longer works with IE 9
+          targets: shouldBuildModern
+            ? isModern
+              ? modernTargets
+              : legacyTargets
+            : undefined,
+          // If users import all core-js they're probably not concerned with
```

Comment:
I think this comment can go. It's left over from a while back. It is removed in `master`.